tcms/signals.py

```diff
@@ -209,19 +209,33 @@ def handle_comments_pre_delete(sender, **kwargs):
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
     """
     Send email updates to assignee after they've been
-    assigned a bug on bug creation.
+    assigned a bug on bug creation or there is a new comment to the bug.
     """
-    if not created or instance.assignee is None:
+    from tcms.core.helpers.comments import get_comments
+
+    comments = get_comments(instance)
```

Comment:
`comment = get_comments(instance).last()` and then work with the object itself. It will either be None or it will have a meaningful value.